### ISSUE TITLE
Tab complete looks hideous...

### ISSUE THREAD

**Issue by djrenren** [chrisbenti/PS-Config#20]
The window that pops up when attempting to tab complete looks absolutely hideous. Is there a way to config this?

**Comment by derekerdmann** [comment]
Tab completion is handled by [PowerTab](http://powertab.codeplex.com/), which may have its own styling capabilities. If you want to disable it, you can comment out the `Import-Module -Name PowerTab` line in the main profile script. I'd like to support configuring which extensions to load, but that's a future release.

**Comment by weswigham** [comment]
Self-modifying powershell profile?

**Comment by chrisbenti** [comment]
ZSH style prompt for config options w/ persisting config file?

**Comment by derekerdmann** [comment]
Yeah, that's a possibility. You'd be able to configure what gets loaded before the main script gets called, but that's all dependent on restructuring the config so that the main script is called from the profile script, similar to Oh My ZSH

**Comment by chrisbenti** [comment]
Also: What should the tab completion look like? Do you prefer the default PowerShell behavior?

**Comment by weswigham** [comment]
What about if tab cycles through completion options, the current one of which is displayed semi-transparently, until you hit right/enter - which completes to the current one and moves the cursor to the end of the completion?

**Comment by chrisbenti** [comment]
Note: This is all (unless you turn it off) is going to require some PowerTab customization (which is supported and I’m ok with). I’d just like to have a good idea of what’s sensible.

As it stands, the current PowerTab config we have enabled handles tab cycling, hitting enter (not right) and then completes the current one). Is the right key something you’d like to use?

The transparency would be really, really awesome. Right now, I find myself having trouble seeing the last lines, which is a problem when I need to use the output from the last command to pick my next action.

**Comment by weswigham** [comment]
So I looked into updating/overriding PowerTab's console list.... and beyond configuring it with provided flags (which allow some moderate customizations), we'd need to package a custom PowerTab to make it happen (or monkeypatch/override some powertab functions). The selection of tab expansion handlers is [hardcoded into a switch](https://github.com/PProvost/PowerTab/blob/810fcc8f6f4d676629aa4c59183a8dd8e57d59d1/TabExpansionLib.ps1#L193) statement in PowerTab's TabExpansionLib.ps1 - not particularly extensible.

**Comment by dragon788** [comment]
As far as "transparency" goes, couldn't you just fade the font to a lighter gray if the default is white, and that shows that it is "suggested" and not "selected"?
